homeworks/viktoriia.sedyk_vikusik96/5-arrays/arrays.js

```diff
@@ -0,0 +1,37 @@
+const defayltArray = [
+    [1, 4, 2, 4, 4, 2, 4, 3, 2, 6, 2, 4, 2, 4, 3, 1, 2, 4, 3],
+    [1, 1, 3, 1, 1, 3, 3, 1, 6, 6, 8, 3, 3, 3, 3, 1, 4, 1, 2],
+    [1, 2, 1, 4, 1, 1, 4, 7, 8, 8, 7, 7, 1, 1, 4, 3, 1, 1, 2],
+    [3, 2, 3, 3, 4, 2, 7, 8, 6, 8, 6, 7, 8, 1, 2, 3, 3, 3, 4],
+    [4, 1, 3, 3, 2, 8, 7, 8, 6, 7, 6, 7, 6, 7, 2, 4, 3, 2, 2],
+    [3, 1, 2, 1, 7, 7, 8, 8, 8, 7, 6, 8, 8, 6, 8, 4, 1, 3, 2],
+    [2, 1, 1, 7, 6, 7, 7, 8, 7, 6, 8, 6, 6, 6, 6, 8, 4, 4, 3],
+    [4, 1, 6, 7, 8, 7, 8, 8, 6, 6, 7, 7, 6, 6, 6, 7, 6, 1, 2],
+    [1, 6, 6, 6, 6, 8, 8, 7, 6, 6, 7, 8, 7, 8, 8, 7, 6, 7, 1],
+    [8, 6, 8, 6, 6, 7, 7, 6, 6, 7, 8, 7, 6, 6, 6, 6, 7, 8, 7],
+];
+
+function convertStrToStarts(array) {
+    return array.map((el) => (el < 5 ? ' ' : '*'));
```

Comment:
У стрілочних функціях можна не писати круглі дужки, окрім тих випадків коли ти повертаєш літерал обʼєкта
```suggestion
    return array.map((el) => el < 5 ? ' ' : '*');
```

Reply:
![image](https://user-images.githubusercontent.com/113469081/209461247-902b9ca5-521e-4f41-b251-6162849ff406.png)
ругаэться eslint без дужок(((

Reply:
Окей, тоді най буде